include/iotconnect_lib.h

```diff
@@ -11,6 +11,8 @@
 
 // SSL style configuration. You can use a different include
 #ifndef ITOTCONNECT_LIB_CONFIG_HEADER
+// FIXME this looks like a typo ITOTCONNECT_LIB_CONFIG_HEADER should be IOTCONNECT_LIB_CONFIG_HEADER
+// however, it's used in multiple repos, so needs thought before modifying?
```

Comment:
I don't think that fixing this typo would affect any other project even if they tracked the latest on the branch. Most projects track a specific commit/label of this repo.